src/PoCompiler.php

```diff
@@ -199,6 +199,7 @@ protected function buildMsgStr(Entry $entry, Header $headers)
             $maxIterations = max($nPlurals, $pluralsFound);
             for ($i = 0; $i < $maxIterations; $i++) {
                 $value = isset($plurals[$i]) ? $plurals[$i] : '';
+                $output .= $entry->isObsolete() ? self::TOKEN_OBSOLETE : '';
```

Comment:
May it need an empty space after `self::TOKEN_OBSOLETE`?

Reply:
I made the same mistake. The TOKEN_OBSOLETE Konstant is:

`const TOKEN_OBSOLETE = '#~ ';`

so it has already a space in it.